lib/primitives/mtx.js

```diff
@@ -2042,6 +2076,22 @@ CoinSelector.MIN_FEE = 10000;
 
 CoinSelector.MAX_FEE = consensus.COIN / 10;
 
+/**
+ * Maximum possible supply of Bitcoin
+ * @const {Amount}
+ * @default
+ */
+
+CoinSelector.MAX_MONEY = 21000000 * 100000000;
```

Comment:
This was fixed when I moved the `CoinSelector` class to `coinselector.js`. Sorry for the confusion.

Reply:
If you end up obsoleting earlier commits in a PR with later commits, it might be nice to rebase the commits to restructure the changes into a minimal set of distinct changes. That would make it easier for reviewers to follow your thought process through your PR.

Reply:
Thanks. I'll keep this in mind for future PRs.